en_US/operations/backup-restore.md

```diff
@@ -37,3 +37,89 @@ Before EMQX 5.0, persistent sessions were stored in memory, and for the EMQX ent
+
+:::
+
+### Export
+
+Data can be exported from any running cluster node.
+
+### Import
+
+To import data, the EMQX node must be running, and certain conditions must be met for a successful import operation:
+
+* If [Core + Replicant](../deploy/cluster/mria-introduction.md#enable-core-replicant-mode) mode is enabled, the data import can only be initiated on core nodes. This does not affect the actual import behavior as the data will be copied to all cluster nodes, both core and replicant. Initiating the import on a core node ensures proper coordination of the operation.
+* Data exported from EMQX Enterprise Edition cluster cannot be imported to EMQX Open Source Edition cluster.
+* The data file must not be renamed.
+
+If any of the above conditions are not satisfied, the import process will be aborted, and an appropriate error message will be displayed.
+
+During the data import operation, the data will either be inserted (if it was not present in the target EMQX cluster) or updated (in case of a conflict). The import process does not remove any existing data from the EMQX cluster.
+
+::: tip Warning
+In rare circumstances, the existing data may not be compatible with the imported data. For example, if the EMQX cluster uses the built-in database authentication mechanism with a 'suffix' salt position, and the imported data defines the same authentication source with a 'prefix' salt position, the old user credentials created before the import will no longer be operational. Therefore, importing data to a non-clean EMQX cluster may require extra caution.
+:::
+
+### Example
+
+This section shows how to import and export data using the command Line Interface.
+
+1. Export data. The file name format of the exported file is `emqx-export-YYYY-MM-DD-HH-mm-ss.sss.tar.gz`, and export directory is `<EMQX data directory>/backup`:
+
+    ```bash
+    $ ./emqx_ctl data export
+    Exporting data to "data/backup/emqx-export-2023-06-19-15-14-19.947.tar.gz"...
+    Exporting cluster configuration...
+    Exporting additional files from EMQX data_dir: "data"...
+    Exporting built-in database...
+    Exporting emqx_admin database table...
+    Exporting emqx_authn_mnesia database table...
+    Exporting emqx_enhanced_authn_scram_mnesia database table...
+    Exporting emqx_app database table...
+    Exporting emqx_acl database table...
+    Exporting emqx_psk database table...
+    Exporting emqx_banned database table...
+    Data has been successfully exported to data/backup/emqx-export-2023-06-19-15-14-19.947.tar.gz.
+    ```
+2. Import data. The name of the imported file should be specified as an absolute path, e.g.:
```

Comment:
@SergeTupchiy I test that relative paths are also work, if you can give priority to the user to use relative paths (this may be simpler).

Reply:
Thanks @wivwiv , we indeed explicitly support relative paths and even a file basename, if it resides in `<EMQX data dir>/backup`.
I've updated it and also added two more examples, hopefully it won't be too wordy..